Uplift/Views/GymDetail/Facilities/CourtView.swift

```diff
@@ -0,0 +1,118 @@
+//
+//  CourtView.swift
+//  Uplift
+//
+//  Created by Phillip OReggio on 11/3/19.
+//  Copyright © 2019 Cornell AppDev. All rights reserved.
+//
+
+import UIKit
+
+class CourtView: UICollectionViewCell {
+
+    // MARK: Gym
+    private var facilityDetail: FacilityDetail!
+    private var dailyGymHours: [DailyGymHours]!
+    private var selectedDayIndex = 0
+
+    // MARK: CollectionView
+    private let courtsCollectionView: UICollectionView
+    private var displayedHours: [FacilityHoursRange] = []
+    private let collectionViewIdentifier = "court"
+
+    // MARK: Display
+    private static let courtSize = CGSize(width: 124, height: 192)
+
+    override init(frame: CGRect) {
+        let flowLayout = UICollectionViewFlowLayout()
+        let spacing: CGFloat = -1 // Overlap Center Court Line
+        flowLayout.scrollDirection = .horizontal
+        flowLayout.minimumInteritemSpacing = 10000
+        flowLayout.minimumLineSpacing = spacing
```

Comment:
lets just do `flowLayout.minimumLineSpacing = -1 // Overlap Center Court Line` instead of putting -1 into a variable since its just being used once